src/lib/pipeline_configuration.py

```diff
@@ -216,7 +217,7 @@ def clean_age_with_range_filter(text):
                     coded_field="constituency_coded",
                     time_field="constituency_time",
                     coda_filename="constituency.json",
-                    analysis_file_key=None,
+                    analysis_file_key="constituency",
                     cleaner=None,
                     code_scheme=CodeSchemes.CONSTITUENCY),
 
```

Comment:
Can we put in some asserts here that the binary scheme in all the follow-up surveys (and demogs) is None?

Reply:
To fix in future pr